app/models/role.rb

```diff
@@ -146,6 +146,7 @@ def self.active_scope(reference_date = Date.current)
     )
   }
 
+  scope :with_ended_readable, -> { unscope(where: [:end_on]).where(end_on: [nil, [Settings.people.ended_roles_readable_for.seconds.ago..]]) }
```

Comment:
Achtung, hiermit wird wieder ein Datum mit Datumzeit verglichen. Würde ich unbedingt anpassen. Und wollen wir das start_on nicht auch rausnehmen, damit künftige Rollen ebenfalls angezeigt werden?

Falls eine Organisation die gelöschten Rollen in alle Ewigkeit sichtbar haben möchte, wäre es wohl einfacher, diesen Fall über ein leeres/nil Setting abzubilden und 0 Tage explizit (und weiterhin in der Default Core Konfiguration) anzugeben.

Reply:
mache hier wieder in to_date und handel den `nil` case wie vorgeschlagen